app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+   
+   get '/tweets' do 
+    if is_logged_in?
+    
+     erb :'/tweets/tweets'
+    else
+    redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+   if is_logged_in?
+    erb :'/tweets/create_tweet'
+   else 
+    redirect '/login'
+    end
+  end
+
+post '/tweets/new' do
+   if !params[:content].empty? 
+      @tweet = current_user.tweets.create(content: params[:content])
```

Comment:
@johnsontaiwo not necessary to use an instance variable here. You can just name this `tweet` and on line 23 just use `redirect to "/tweets/#{tweet.id}"`